src/typescript/frontend/src/app/test/route.ts

```diff
@@ -0,0 +1,10 @@
+import { getAptos } from "lib/utils/aptos-client";
+import { NextResponse } from "next/server";
+
+export const revalidate = 2;
```

Comment:
Not a big deal with just a single test but because we'll be using varying cache times in different tests you'll probably have to abstract some of the data passed in to them to a function, and you don't want to pass magic numbers to those functions

Ideally we export the expected test's revalidation time as a constant somewhere and then get it from there, just like how we do `REVALIDATION_TIME`